ReText/editor.py

```diff
@@ -197,6 +197,9 @@ def contextMenuEvent(self, event):
 		actions = [self.parent.act(sug, trig=self.fixWord(sug)) for sug in suggestions]
 		menu = self.createStandardContextMenu()
 		menu.insertSeparator(menu.actions()[0])
+		menu.insertAction(menu.actions()[0], self.parent.actionMoveDown)
+		menu.insertAction(menu.actions()[0], self.parent.actionMoveUp)
+		menu.insertSeparator(menu.actions()[0])
```

Comment:
I used `insert*` instead of `add*` to match the other lines in this method. Should we use `add*` for all of them ? 
Also I can't move those lines up as the menu is not created yet. Or we should not use the default menu creation at all  in this case.

Reply:
`insert*(menu.actions()[0], ...)` adds an action top top, `add*(...)` adds to bottom. In my opinion spell checker actions are popular and should be on top, these ones are less popular and can be on bottom.

What do you mean by “not use the default menu creation at all”?

I think the solution is like this:
- Always create the menu on top of this function, always append the move up/down actions;
- Replace `return QTextEdit.contextMenuEvent(self, event)` with `menu.exec(event.globalPos())`.

Reply:
Done. I kinda rewrote the method to add the action even if we don't need the spellcheck, while keeping the spellcheck options on top, and the move at the bottom